libs/designer/src/lib/ui/panel/templatePanel/quickViewPanel/quickViewPanel.tsx

```diff
@@ -8,14 +8,23 @@ import { TemplatesPanelContent } from '@microsoft/designer-ui';
 import { getQuickViewTabs } from '../../../../core/templates/utils/helper';
 import Markdown from 'react-markdown';
 
+export interface QuickViewTabProps {
+  templateId: string;
+  workflowAppName: string;
+}
+
 export const QuickViewPanel = () => {
   const dispatch = useDispatch<AppDispatch>();
   const intl = useIntl();
-  const { manifest } = useSelector((state: RootState) => ({
+  const { manifest, templateName, resourceGroup } = useSelector((state: RootState) => ({
     manifest: state.template.manifest,
     templateName: state.template.templateName,
+    resourceGroup: state.workflow.resourceGroup,
   }));
-  const panelTabs = getQuickViewTabs(intl, dispatch);
+  const panelTabs = getQuickViewTabs(intl, dispatch, {
+    templateId: templateName ?? '',
+    workflowAppName: resourceGroup,
```

Comment:
A Workflow App is what you choose in the portal to open the workflows list blade... resource group is a logical container in a subscription in Azure which contains all resources, logging resource group is not going to be helpful at all here.

As we have talked yesterday we need the app name logic app name...